openverse_catalog/dags/providers/provider_api_scripts/science_museum.py

```diff
@@ -171,19 +173,28 @@ def _get_creator_info(obj_attr):
     return creator_info
 
 
+def _get_filesize(image_url):
+    """
+    Get the size of the image in bytes.
+    """
+    resp = delayed_requester.get(image_url)
+    if resp:
+        filesize = int(resp.headers.get("Content-Length", 0))
+        return filesize if filesize != 0 else None
```

Comment:
If I'm understanding this correctly, this will make a request for every image we receive from their API. Should this maybe go the way of NYLP and not report `filesize` since it's not provided by their API? And if not, should this maybe be doing a `HEAD` request rather than a full `GET`?

Reply:
I agree with not reporting `filesize` at all, this is a more sustainable approach.
By the way, I was sure I was using the `HEAD` requests everywhere, and not `GET`'s 😵‍💫